support/netutil/netutil.go

```diff
@@ -72,3 +72,64 @@ func watchSignalWithinTimeout(timeout time.Duration, errSignal chan struct{}, ac
 	}
 	action()
 }
+
+// get the local area network IP address
+func GetLocalHost() string {
```

Comment:
You are getting the local public IPs I would expect this method to return a list of string

I wan have eth0, tun0 and wlan0

You will return only the first one, so you cannot bind on the second or later ones

Reply:
Yes, it currently supports passing in ipv4, ipv6 to get the corresponding list of IPs. Thank you.